src/main/java/mate/academy/lib/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.lib;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "Dimension3306");
+
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't load JDBC driver", e);
+        }
+    }
+
+    public static Connection getConnection() throws SQLException {
+        return DriverManager.getConnection(DB_URL, DB_PROPERTIES);
+    }
```

Comment:
Don't forget to close the `Connection` object after usage to prevent resource leaks. This can be done using try-with-resources or by calling `close()` in a finally block.